### ISSUE TITLE
Done/Next button on mobile keyboard not working

### ISSUE THREAD

**Issue by razilzul** [YousicianGit/UnityNativeEdit#50]
Tried the latest release and build from scratch using the demo scene included with Unity 2017 and am unable to close keyboard when pressing on Done/Next button no matter what configurations i tried in the NativeEditBox inspector. Nothing happens when i click it and the only way for me to close the mobile keyboard is to press the back button on the device. Do you know whats causing the problem? Tried on both Unity 2017 and 5.6.3. Devices tested was on S6 and Xiaomi Mi Max. Also tried including the manifest and without as stated in a previous issue that it is no longer required with the same results. Other than that big issue everything else is working great!

**Comment by Hemaolle** [comment]
You should get the "On Return Pressed" event when the Done/Next button is pressed. You can hook it up in Unity with the inspector settings for the `NativeEditBox` component. You seem to be right in that the demo scene doesn't have anything hooked up to these events so nothing will happen.

**Comment by razilzul** [comment]
Thank you it works now! all i had to do was to add `NativeEditBox.SetFocus` to false on the `On Return Pressed` event in `NativeEditBox` component. I believe this should be the default no? Anyway thank you so much for the help and great plugin!

**Comment by Hemaolle** [comment]
Yeah, probably would make sense to do that if no custom behavior is added. I wonder if it would make sense to do something like check if there are eventlisteners attached when firing the event. Or perhaps a boolean flag would be more explicit. Feel free to implement and submit a pull request :slightly_smiling_face:
